backend/src/main/java/com/capstone/moneytree/model/OrderType.java

```diff
@@ -0,0 +1,8 @@
+package com.capstone.moneytree.model;
+
+public enum OrderType {
```

Comment:
We might want to change this in the future to split the "side" (buy/sell) and the "type" (market, limit, stop, ...). It would allow easier communication with the api and easier access to atomic order info. See https://alpaca.markets/docs/api-documentation/api-v2/orders/